docs/source/start.rst

```diff
@@ -78,14 +78,25 @@ available commands in ASReview LAB, type :code:`asreview lab --help`.
 	techniques, and query strategies). Use an integer between 0 and 2^32 - 1.
 
 
+Set environment variables
+-------------------------
+
 The following evironment variables are available.
 
 .. option:: ASREVIEW_PATH
 
-	The path to the folder with project. Default `~/.asreview`.
+	The path to the folder with project. Default `~/.asreview`, or search the location in Windows with 
+
+  .. code:: bash
+
+  dir "\.asreview"/ s
+
+  In MacOS or Linux operating systems use
+
+  .. code:: bash
+
+  find / -type d -name ".asreview"
```

Comment:
```suggestion
	The path to the folder with project. Default `~/.asreview`.
```

To move this forward, I propose to remove this part of the pull request. The main point of discussion is whether this is the right location to provide this information. This line is on setting the envs and not finding the project folder. 

Added some information to the discussion platform https://github.com/asreview/asreview/discussions/1203

Reply:
nice, much better place to store this information